internal/pbio/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/bsm/feedx/internal/pbio
```

Comment:
sorry, not a fan, we can put this in https://github.com/bsm/pbio/

Reply:
hmm, let me take a look at this, I have an "idea"

Reply:
> sorry, not a fan, we can put this in https://github.com/bsm/pbio/

That's the intent, just wanted to test it quick in feedx

Reply:
https://github.com/bsm/pbio/pull/2/files

Reply:
https://github.com/bsm/pbio/releases/tag/v0.2.2